src/helpers/notifications.ts

```diff
@@ -4,37 +4,39 @@ import { NavigationService } from 'app/services';
 
 const i18n = require('../../loc');
 
-export const getOnboardingAddEmailParams = (): AddNotificationEmailParams => ({
-  onSkipSuccess: () => {
-    CreateMessage({
-      title: i18n.contactCreate.successTitle,
-      description: i18n.onboarding.successCompletedDescription,
-      type: MessageType.success,
-      buttonProps: {
-        title: i18n.onboarding.successCompletedButton,
-        onPress: () => {
-          NavigationService.navigate(Route.MainTabStackNavigator, { screen: Route.Dashboard });
+export const getOnboardingAddEmailParams = (): AddNotificationEmailParams => {
+  return {
```

Comment:
The return here is not necessary - it's arrow function, isn't it? :)

Reply:
yeah I was console.logging there :D